internal/bminventory/inventory.go

```diff
@@ -458,6 +461,7 @@ func (b *bareMetalInventory) RegisterClusterInternal(
 			HTTPSProxy:               swag.StringValue(params.NewClusterParams.HTTPSProxy),
 			NoProxy:                  swag.StringValue(params.NewClusterParams.NoProxy),
 			VipDhcpAllocation:        params.NewClusterParams.VipDhcpAllocation,
+			NetworkType:              swag.StringValue(params.NewClusterParams.NetworkType),
```

Comment:
We should validate that the provided network type is valid - either OpenshiftSDN , or OVNKubernetes

Reply:
this ?
https://github.com/openshift/assisted-service/pull/2113/files#diff-72619caf255df33ccaa77f4a5ea4b9980c41cea8fbbbde96f8c62e9085282195R391

Reply:
This is Register and the link above is for update